client/src/components/buttonSelector/buttonSelector/ButtonSelector.scss

```diff
@@ -1,6 +1,9 @@
 @import '../../../scssStyles/mixins';
 .button-selector-items {
   display: flex;
-  justify-content: space-around;
```

Comment:
Don't remove styles from other components
It may break other pages using this component!

Reply:
You should wrap this, or apply styles/className prop to the component

Reply:
Added back in, used boolean props 'scroll' and 'isScheduleComponent' to prevent the problem of breaking other pages currently using the component.